README.md

```diff
@@ -215,7 +215,6 @@ questions and get our support.
 
 ## Links
 
-- [Computer Vision Annotation Tool (CVAT) has a New Home](https://opencv.org/opencv-and-cvat-computer-vision-annotation-tool-intel/)
```

Comment:
@mdacoca , again. The change is mine. How have you removed it?

Reply:
When I create a branch from dev it downloads a lot of extra stuff including some things from other branches and commits (including some of my commits and some not mine). So some of them are reverted some not. I try to figure out why it is this way.

Reply:
@nmanovic I will redo it completly, sorry for the mess